Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,28 @@
+class LibcdioParanoia < Formula
+  desc "CD paranoia on top of libcdio"
+  homepage "https://github.com/rocky/libcdio-paranoia"
+  url "https://github.com/rocky/libcdio-paranoia/archive/refs/tags/release-10.2+2.0.1.tar.gz"
+  version "10.2+2.0.1"
+  sha256 "7a4e257c85f3f84129cca55cd097c397364c7a6f79b9701bbc593b13bd59eb95"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libcdio"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", *std_configure_args,
+                          "--without-versioned-libs",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  test do
+    assert_match(/cdparanoia/, shell_output("#{bin}/cd-paranoia -V 2>&1").partition(" ").first)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.